solana-programs/anchor/audius-data/scripts/install-dev-dependencies.sh

```diff
@@ -34,6 +34,9 @@ yarn install
 # init solana keypair
 solana-keygen new --no-bip39-passphrase --force -o "$HOME/.config/solana/id.json"
 
+# airdrop funds
+solana airdrop 1000000
```

Comment:
when i was pairing with vicky she had trouble from a fresh install without airdropping funds, so here we do that to facilitate local dev